dubbo-rpc/dubbo-rpc-dubbo/src/main/java/org/apache/dubbo/rpc/protocol/dubbo/telnet/InvokeTelnetHandler.java

```diff
@@ -240,10 +128,96 @@ public String telnet(Channel channel, String message) {
         return buf.toString();
     }
 
+
     private boolean isServiceMatch(String service, ProviderModel provider) {
         return provider.getServiceName().equalsIgnoreCase(service)
                 || provider.getServiceInterfaceClass().getSimpleName().equalsIgnoreCase(service)
                 || provider.getServiceInterfaceClass().getName().equalsIgnoreCase(service)
                 || StringUtils.isEmpty(service);
     }
+
+    private List<Method> findSameSignatureMethod(List<ProviderMethodModel> methods, String lookupMethodName, List<Object> args) {
+        List<Method> sameSignatureMethods = new ArrayList<>();
+        for (ProviderMethodModel model : methods) {
```

Comment:
can be rewritten as 
`return methods.stream.filter(method->method.getName().equals(lookupMethodName) && method.getParameterTypes().length == args.size()).collect(Collectors.toList());`

Reply:
I am not sure if I can use stream api 。